tests/test_package_generation.py

```diff
@@ -129,6 +129,7 @@ def test_optional_funder(
     tmp_path: pathlib.Path, generate_package: typing.Callable, funder: str
 ) -> None:
     """Test package generation."""
+    """Test specifying funder or not in package generation."""
```

Comment:
I assume you didn't mean to have two docstrings?

Reply:
Nope! Now fixed thanks